src/main/java/com/minecolonies/rankup/modules/magibridge/config/MagibridgeConfig.java

```diff
@@ -10,8 +11,8 @@
 public class MagibridgeConfig
 {
 
-    @Setting(value = "rankup-Message", comment = "Message to discord when a player ranks up! \n valid entries: {player}, {next_group} and {current_group}")
-    public String rankupMessage = "Player {player} has left the {current_group} group and joined the {next_group} group! :smile:";
+    @Setting(value = "rankup-Message", comment = "Message to discord when a player ranks up! \n valid entries: " + Constants.PlayerInfo.PLAYER_NAME + ", " + Constants.ModuleInfo.MAGIBRIDGE_NEXT + " and " + Constants.ModuleInfo.MAGIBRIDGE_CURR + "")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 248 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00103)